modules/renter/contractor/contractor.go

```diff
@@ -133,6 +134,7 @@ func newContractor(cs consensusSet, w wallet, tp transactionPool, hdb hostDB, p
 
 		cachedRevisions: make(map[types.FileContractID]cachedRevision),
 		contracts:       make(map[types.FileContractID]modules.RenterContract),
+		downloaders:     make(map[types.FileContractID]*hostDownloader),
```

Comment:
ok. I'm gonna leave it for now because the change is more involved than you might think (a lot of downloader code only has access to to the ID, not the host key). I do agree that we should move this field (and possibly a few others) to using the host key.